src/QsCompiler/LlvmBindings/Interop/LLVMMemoryBufferRef.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation and Contributors. All Rights Reserved. Licensed under the MIT License (MIT).
```

Comment:
(Nit) FYI: In this file and below (in this PR's diff) the syntax highlighting is active, but for the files above the syntax highlighting is NOT active. If for you it is the same then in those files (with no syntax highlighting) there can be something hidden (file encoding?) that prevents the syntax highlighting to work properly (in this PR's diff). And there is a chance that if they are merged to main, then in subsequent PRs the syntax highlighting can suffer too.

Reply:
Hmm, that's odd. The syntax highlighting shows up fine on my screen, so it could be an issue with your browser window?

Reply:
Yep, the diff is large/long, that could be just a hiccup in my browser.